.editorconfig

```diff
@@ -1,9 +1,9 @@
 
 [*]
 charset = utf-8
-end_of_line = crlf
+end_of_line = lf
```

Comment:
That's a dangerous change

Reply:
We're already using LF in gitattributes ;)